src/.eslintrc

```diff
@@ -0,0 +1,94 @@
+// .eslintrc
```

Comment:
I am pretty sure that .eslint files need to be in root folder(@MrShtrahman )

also, both of you start with an M and it bummer , i will not tell you to change but im Seriously upset

Reply:
You actually think i'm going to change "MrShtrahman" becuase you're too lazy **TO TYPE ANOTHER KEYBOARD???**

Reply:
But yeah, good point about the files being transferred to the root directory.